tests/test_forcing_profiles_extraction.py

```diff
@@ -6,9 +6,17 @@
 from lagtraj.utils import validation
 
 
-@pytest.mark.parametrize("gradient_method", ["boundary", "regression"])
+@pytest.mark.parametrize(
```

Comment:
Great! 🌟 🌟 🌟

Reply:
It is nice that pytest allows for these combinations.